File: SongsWebApp/SongsBurger/ClientApp/app/Models/ExportText.ts
export interface IExportText {
    format: string;
    formattedText: string;
}

export class ExportText implements IExportText {
    static fromData(data: IExportText) {
        return new this(
            data.format,
            data.formattedText);
    }

    constructor(
        public format: string = "",
        public formattedText: string = "") {

    }
}

File: SongsWebApp/SongsApi/Controllers/RequestsResponses/ExportText.cs
namespace SongsApi.Controllers.RequestsResponses
{
    public class ExportText
    {
        public string Format { get; set; }
        public string FormattedText { get; set; }

        public ExportText()
        {
            Format = string.Empty;
            FormattedText = string.Empty;
        }
    }
}
